presto-kafka/src/main/java/io/prestosql/plugin/kafka/encoder/json/format/SecondsFormatFunctions.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.kafka.encoder.json.format;
+
+import java.util.function.Function;
+
+import static io.prestosql.plugin.kafka.encoder.json.format.JsonFormatFunctions.millisToSeconds;
+import static io.prestosql.spi.type.DateTimeEncoding.unpackMillisUtc;
+
+public final class SecondsFormatFunctions
+{
+    private SecondsFormatFunctions() {}
+
+    public static Function<Long, String> formatSecondsFunc()
+    {
+        return millis -> String.valueOf(millisToSeconds(millis));
+    }
+
+    public static Function<Long, String> formatSecondsWithTZFunc()
```

Comment:
Another situation in which we're throwing away tz info.

Reply:
This is like unix `time()`, it's fine.